docs/src/tutorial/temporal_resolution.md

```diff
@@ -0,0 +1,175 @@
+# Temporal Resolution Tutorial
+
+Welcome to Spine Toolbox's Temporal Resolution tutorial.
+
+This tutorial provides a step-by-step guide to include uniform and/or flexible temporal resolution in your model.
+
+For more information on how time works in SpineOpt, see the [Temporal Framework](https://spine-tools.github.io/SpineOpt.jl/latest/advanced_concepts/temporal_framework/) documentation.
+
+## Introduction
+
+### Installation and upgrades
+
+If you haven't yet installed the tools yet, please follow the installation guides: 
+- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
+- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+
+If you are not sure whether you have the latest version, please upgrade to ensure compatibility with this guide.
+
+- For Spine Toolbox:
+    - If installed with pipx, then use `python -m pipx upgrade spinetoolbox`
+    - If installed from sources using git, then `git pull`, `python -m pip install -U -r requirements.txt`
+
+- For SpineOpt: [SpineOpt upgrade guide](https://github.com/spine-tools/SpineOpt.jl#upgrading)
+
+### Model assumptions 
+
+- This tutorial builds upon the Simple System tutorial. Please follow [that tutorial](https://spine-tools.github.io/SpineOpt.jl/latest/tutorial/simple_system/) first, or download the [finished system](**NEEDS LINK**).
+- The scenario runs for 5 hours and the default temporal resolution is 1 hour.
+- The fuel_node has a temporal resolution of [1, 1, 1, 2] hours.
+
+## Tutorial
+
+### Adjustments to Simple System
+
+Before demonstrating flexible temporal resolution, we need to adjust the Simple System tutorial so the results are more interesting to examine.
+
+!!! note If you are ever missing a window in the Spine DB Editor, go to the *Hamburger menu* (top right) > under View: Docks > Choose the missing window - then drag and resize the window to your preference
+
+#### Adding variable electricity demand
+
+In SpineToolbox, double click on the *InputData*  block.
```

Comment:
Yes I talked to Diego about this. It depends whether we give them the entire project or just the JSON. If it's the JSON I'll need to add a bit about setting up the workflow and importing the JSON.

Reply:
We'll provide the JSON file as that is more github friendly. The reason why I haven't put them on there yet is because I'm also coupling unit tests to the JSON files but I can't get the tests to work .... yet.

Reply:
I updated the instructions to work with a JSON file.